Security/Firewall/GetJWTListener.php

```diff
@@ -98,26 +97,34 @@ public function handle(GetResponseEvent $event)
     {
         $request = $event->getRequest();
 
-        if ($this->options['post_only'] && !$request->isMethod('POST')) {
-            $event->setResponse(new JsonResponse('invalid method', 405));
+        if (!in_array($request->getMethod(), ['POST', 'GET'])) {
```

Comment:
This check should be removed, since we're letting the Symfony Router handle the allowed methods.

But if we're not removing this, then we need to throw a `Symfony\Component\Security\Core\Exception\AuthenticationException`.